docs/how-to-guides/upgrade-openebs.md

```diff
@@ -0,0 +1,279 @@
+# Upgrading and migrating OpenEBS data plane components
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Step 1: Upgrade OpenEBS control plane components](#step-1-upgrade-openebs-control-plane-components)
+* [Step 2: Upgrade OpenEBS data plane components](#step-2-upgrade-openebs-data-plane-components)
+	* [Upgrade cStor pools](#upgrade-cstor-pools)
+	* [Upgrade cStor volumes](#upgrade-cstor-volumes)
+* [Step 3: Verify](#step-3-verify)
+* [Summary](#summary)
+* [Additional resources](#additional-resources)
+
+## Introduction
+
+This guide provides steps to upgrade OpenEBS dataplane components from `v1.12.0`
+to `v2.2.0` and migrate from cStor Pools/Volumes to latest CSI based
+provisioning (CSPC Pools/Volumes).
+
+## Prerequisites
+
+To upgrade the OpenEBS data plane components, we need the following:
+
+* Lokomotive `v0.6.0`
+    ```bash
+    lokoctl version
+    v0.6.0
+    ```
+
+* A Lokomotive cluster accessible via `kubectl` deployed on a
+  [Equinix metal](../configuration-reference/platforms/packet.md) or
+  [Bare metal](../configuration-reference/platforms/baremetal.md).
```

Comment:
What if I deploy `OpenEBS` component to non-Lokomotive cluster? :smile:

Reply:
This guide is not for non Lokomotive clusters.

Reply:
Hmm, why? I think we can just say `openebs` component installed. As far as I know, it's completely fine to use components on non-Lokomotive clusters.